docs/tutorials/configure/pet-photographer.md

```diff
@@ -0,0 +1,267 @@
+1. Choose `Data Management` as the type and name your instance of the data manager `dm`.
+   This service syncs data from your robot to the Viam app in the cloud.
+1. Select **Create**.
+1. On the panel that appears, you can manage the capturing and syncing functions individually.
+   By default, the data management service captures data every 0.1 minutes in the <file>~/.viam/capture</file> directory.
+
+   You can leave the default settings as they are.
+   Click **Save Config** at the bottom of the window.
+
+   ![An instance of the data management service named "dm". The cloud sync and capturing options are toggled on and the directory is empty. The interval is set to 0.1](/tutorials/pet-photographer/data-management-services.png)
+
+   For more detailed information see [Add the data management service](/services/data/configure-data-capture/#add-the-data-management-service).
+   {{% /tab %}}
+   {{% tab name="JSON Template" %}}
+   Add the vision service object to the services array in your rover’s raw JSON configuration:
+
+```json {class="line-numbers linkable-line-numbers"}
+    {
+      "name": "dm",
+      "type": "data_manager",
+      "namespace": "rdk",
+      "attributes": {
+        "sync_interval_mins": 0.1,
+        "capture_dir": "",
+        "tags": [],
+        "additional_sync_paths": []
+      }
+    },
+  ... // Vision and other services
+```
+
+Click **Save Config** and head back to the **Builder** mode.
+
+{{% /tab %}}
+{{< /tabs >}}
+
+## Configure the color filter camera
+
+Before you can interact with the vision and data management services, you must configure your camera to filter color and store photos to Viam's cloud.
+
+### Configure the module's executable
+
+This tutorial makes use of [Viam's color filter example](https://github.com/viam-labs/modular-filter-examples/tree/main/colorfilter). However, you can modify the filter's source code or write your own filter for different components using [this guide](/extend/modular-resources/create/).
+
+Navigate to the `modular-filter-examples/colorfilter/module` directory on your robot's terminal and run the following command:
+
+```{class="command-line" data-prompt="$"}
+go build
+```
+
+This command compiles the source code in the colorfilter/module directory and generate an executable with the same name as the module, which is 'colorfilter'.
+
+{{< alert title="Tip" color="tip" >}}
+Go to the <file>colorfilter/module</file> directory within the color filter module you cloned and get the absolute path to your `colorfilter` module for later use by running:
+
+```sh {class="command-line" data-prompt="$"}
+realpath colorfilter
+```
+
+{{< /alert >}}
+
+Read [Configure a local module](https://docs.viam.com/extend/modular-resources/configure/#configure-a-local-module) for more information on configuring a local module.
+
+### Add local module
+
+Select the **Modules** subtab in the **Config** panel to upload the local color filter module to your robot's system in the Viam app.
```

Comment:
I don't think this uploads anything! You're telling viam (configuring) where the executable it needs to run is, so that it can run it:
```suggestion
Select the **Modules** subtab in the **Config** panel to configure the local color filter module for your robot's system in the Viam app.
```